data/gegno/gegno intro missions.txt

```diff
@@ -0,0 +1,278 @@
+				
+			label search
+			`	Continuing on your journey, a vague radar signature similar to the ones you saw earlier, though more erratic, appears a bit further out. As you approach it, a large arch-like structure becomes visible in the distance, but disappears shortly after. You set your ship down close by and ready yourself as you exit it once again to investigate.`
+			`	Much to your surprise, there is a large humanoid figure standing where the signature originated from. You were sure your radar scans showed no life form signature, but standing in front of you is a large, bulky being cladded in what looks like a form of metallic armor, wielding some type of spear-like weapon. The figure moves around attentively as if it is intently focused on its surroundings, but appears to take no interest in your presence. Unlike other aliens you have met in the galaxy, this encounter feels very odd in that there is no immediate interaction with you.`
+			choice
+				`	Approach the alien.`
+				`	Turn back to your ship.`
+			`	All of a sudden, the ground starts to vibrate, rapidly increasing in intensity with every second. You take out your sidearm in a panic, but as you look back up at the alien figure, you notice that they. too, enter a defensive stance- but not towards you. A large maw bursts open in the sand to your left, and out of it comes a massive creature covered in spikes and scales, hundreds of times the size of you. Taking aim with your sidearm, you barely have time to react as it plummets back into the sand over beyond your right, but get a quick glimpse of the alien creature riding it down into the desert as they impale it with their weapon. Unsure of what comes next during the omnious silence, you slowly start to back your way towards your ship.`
+			choice
+				`	Leave the planet immediately.`
+					goto leave
+				`	Distance yourself from the site but continue to watch.`
+					goto watch
+				`	Try to help the alien defeat the beast.`
+					goto help
+					
+			label help
+			`	Despite being in a first contact situation without a proper interaction, you decide to attempt assisting the alien and aim your sidearm at the beast, preparing to fire. Instead of digging back under the surface, the beast continues to thrash about where it is while the alien batters it down with their spear weapon. Making sure you don't accidentally hit them, you hold your breath and wait until the right moment comes, aiming for the creature's head. Everything becomes tense as the space between you and the flailing creature diminishes, and your pulse begins to race. A gust of sand is forced directly towards you, and as it clears by that's when you fire your weapon. A laser shot flies directly into the face of the beast.`
```

Comment:
```suggestion
			`	Despite being in a first contact situation without a proper interaction, you decide to attempt assisting the alien and aim your sidearm at the beast, preparing to fire. Instead of digging back under the surface, the beast continues to thrash about where it is while the alien batters it down with their spear weapon. Making sure you don't accidentally hit them, you hold your breath and wait until the right moment comes, aiming for the creature's head. Everything becomes tense as the space between you and the flailing creature diminishes, and your pulse begins to race. A gust of sand blasts directly towards you, and once it clears you fire your weapon. A laser shot flies directly into the face of the beast.`
```